release.gradle

```diff
@@ -5,15 +5,15 @@ def isReleaseBuild() {
     return VERSION_NAME.contains("SNAPSHOT") == false
 }
 
-def getRepositoryUrl() {
+def getMavenRepositoryUrl() {
```

Comment:
What's this change about?  Seems unrelated (though not necessarily wrong, I'd prefer to see it in a separate PR for bookkeeping sake).